testing/crypto++/APKBUILD

```diff
@@ -4,33 +4,36 @@ pkgname=crypto++
 _pkgname=cryptopp
 pkgver=5.6.5
```

Comment:
This version is fairly old now.

https://www.cryptopp.com/downloads.html#CRYPTOPP_820